### ISSUE TITLE
Improve documentation at the function level (docstrings)

### ISSUE THREAD

**Issue by gidili** [MetaCell/minian#51]

**Comment by gidili** [comment]
The goal of this is to improve string docs so that we can setup "read the docs" or similar

**Comment by zsinnema** [comment]
yes we can use sphinx for that with a RTD theme (https://sphinx-rtd-theme.readthedocs.io/en/stable/)

**Comment by zsinnema** [comment]
[html.zip](https://github.com/MetaCell/minian/files/4823878/html.zip)

I've added a docstring template (google python docstring format) to all functions. The docstring contains the parameters of the fuctions.

**Comment by gidili** [comment]
This work enables #25
